libvips/colour/scRGB2sRGB.c

```diff
@@ -140,7 +140,7 @@ vips_scRGB2sRGB_line_16(unsigned short *restrict q, float *restrict p,
 		q += 3;
 
 		for (j = 0; j < extra_bands; j++)
-			q[j] = VIPS_CLIP(0, (int) (p[j] * 256.0), USHRT_MAX);
+			q[j] = VIPS_CLIP(0, (int) (p[j] * 65535.0), USHRT_MAX);
```

Comment:
Looks like `XYZ2scRGB.c` and `scRGB2XYZ.c` will need revising too, since they currently don't change the alpha range.

Reply:
So does `scRGB2BW`, my bad